test/devnet-restart.test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from "chai";
+import { starknet } from "hardhat";
+
+import { TIMEOUT } from "./constants";
+
+describe.only("Devnet restart", function() {
+    this.timeout(TIMEOUT);
+
+    const salt = "0x99";
+
+    it("should pass", async () => {
+        try {
+            const response = await starknet.devnet.restart();
+            expect(response).to.be.undefined;
+        } catch (error) {
```

Comment:
Why catch this? If it throws, it already makes the test fail. By failing like this, the real failure cause is hidden.

Reply:
[d9788ec](https://github.com/Shard-Labs/starknet-hardhat-example/pull/49/commits/d9788ec7f843ac9bb798c14199a7501c5516ea95)